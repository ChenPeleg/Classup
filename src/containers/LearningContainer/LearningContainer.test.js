//https://medium.com/better-programming/how-to-test-your-react-container-components-fd71071cc3e0
