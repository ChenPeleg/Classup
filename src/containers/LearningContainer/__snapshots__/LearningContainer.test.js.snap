// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LearningContainer> general rendering renders correctly 1`] = `
<ContextProvider
  value={
    Object {
      "info_questions": Array [],
      " questionNext": 1,
      " questionNumber": 8,
      "totalQuestions": 2,
    }
  }
>
  <ProgressBar
    viewAnotherQuestionHandler={[Function]}
  />
  <SummaryContainer
    resetHandler={[Function]}
    sumData={
      Array [
        Array [],
        Array [],
        Array [],
      ]
    }
  />
</ContextProvider>
`;
